scripts/mypuush

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+##############################################################################
+###                                                                        ###
+###  File : mypuush                                                        ###
+###  Language : bash                                                       ###
+###  Description : takes a screenshot of your screen, uploads the file     ###
+###     and send the link it the clipboard                                 ###
+###                                                                        ###
+###  Dependencies : xclip, jq, gnome-screenshot                            ###
+###                                                                        ###
+###  Author : Raphael Ehret                                                ###
+###  E-mail : raphael.ehret@gmail.com                                      ###
+###                                                                        ###
+##############################################################################
+
+
+print_help () {
+
+    name=$(basename $0)
+
+    echo "$name takes a screenshot, uploads it and sends the link in your clipboard"
+    echo "Usage : $name [option] [-a|-w]"
+    echo ""
+    echo "$name [option] -a"
```

Comment:
The intended behavior when using no option is different from the area and window option (full desktop screenshot). I don't get what you mean by Use `{(a|area),(w|window)}`.

Code injection is prevented by the getopt case below